### DISCUSSION TITLE
Passing filters through url parameters

### DISCUSSION THREAD

**Discussion by zachjamesjohn** [microsoft-search/pnp-modern-search#3177] [General]
Is it possible to pass the filters to a new page through url parameters like the query is passed? 

In this case I know I would need a search results web part on the first page to populate the filters. I would end up hiding this search results part.

The design I am trying to reproduce has the search bar and filters on one page that send the query to a second page displaying the results. It seems this is not currently possible.

**Comment by kasperbolarsen** [comment]
Out of the box I think the answer is no. I have a hard time trying to understand the reason for the ux choice. The current combos are based on the concept that the end users can modify the search query and apply different filters as they see the results and gets additional ideas about what exactly they are looking for.
I guess you have a special use case, so we might be able to point out some options if you can share some details?

**Comment by zachjamesjohn** [comment]
Thanks for the response. 

The design I was given to reproduce has the search bar and the filters on the first page and a second page with the search bar, filters and results 

From what I gather it is purely for aesthetic reasons. They want some other features other then the pnp modern search parts on the first page (home page) and want the second page to be only search related.

It seems that this design they want is not possible. Other then this the search on the second page works great. I think I will just push back on the design choice and see if they will budge.

Thanks
